includes/core/server/class-request.php

```diff
@@ -663,4 +678,64 @@ public function validate_user_name( string $user_name ) {
 			);
 		}
 	}
+
+	/**
+	 * Validates API endpoint or route.
+	 *
+	 * @param string $api_route API route to validate.
+	 *
+	 * @throws Invalid_Request_Parameter_Exception
+	 */
+	public function validate_api_route( string $api_route ) {
+
+		// I will use array keys check validate is request to improve performance. Maybe we can move this outside of this function.
```

Comment:
Can move this out as mentioned and remove this comment

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/6234/commits/ad6dc07e9c994c0c3a97925a0350f54ab67b58e5